### ISSUE TITLE
New Term: prominent axillary crease

### ISSUE THREAD

**Issue by CorneliusDreyling** [obophenotype/human-phenotype-ontology#10569]
**Preferred term label:**
Prominent axillary crease

**Synonyms**
Double axillary creases

**Definition (free text, please give PubMed ID)**
Prominent or even doubled axillary creases observed in neuromuscular diseases (e.g. FSHD) due to wasting of the pectoral muscles. (doi: 10.1212/CON.0000000000000399)
**Parent term (use hpo.jax.org/app)**
Pectoral muscle hypoplasia/aplasia HP:0005258
**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:269
**Your nano-attribution (ORCID)**
0009-0004-9942-3886

**Comment by pnrobinson** [comment]
@CorneliusDreyling 
It seems it would be better to add the "Prominent axillary crease" as part of the description of wasting of pectoral muscles rather than an independent phenotype. This would be better unless there are situations without wasting of pectoral muscles that are also characterized by "Prominent axillary crease" .

**Comment by CorneliusDreyling** [comment]
@pnrobinson 
Indeed, the wasting of the lateral muscles is essential. The addition of "prominent axillary creases", or "double axillary creases", in description of pectoral muscle wasting, appears to be a reasonable approach.

**Comment by pnrobinson** [comment]
Thanks!
I have updated the definition to 

Pectoralis amyotrophy
HP:0012037
Wasting of the pectoral muscles, i.e., of the pectoralis major and pectoralis minor. This finding is often manifested by prominent axillary creases or double axillary creases.
